.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,45 @@
 ### Description of work
 
-#### Summary of work
-<!-- Please provide a short, high level description of the work that was done.
+<!-- Please provide an outline and reasoning for the work.
+If there is no linked issue provide context.
 -->
 
-<!-- Why has this work been done? If there is no linked issue please provide appropriate context for this work.
-#### Purpose of work
-This can be removed if a github issue is referenced below
--->
-
-Fixes #xxxx. <!-- and fix #xxxx or close #xxxx xor resolves #xxxx. One line per issue fixed. -->
-<!-- alternative
-*There is no associated issue.*
--->
+Closes #xxxx. <!-- One line per closed issue. -->
 
-<!-- If the original issue was raised by a user they should be named here. Do not leak email addresses
+<!-- If issue raised by user. Do not leak email addresses.
 **Report to:** [user name]
 -->
 
-#### Further detail of work
-<!-- Please provide a more detailed description of the work that has been undertaken.
--->
-
 ### To test:
 
-<!-- Instructions for testing.
-There should be sufficient instructions for someone unfamiliar with the application to test - unless a specific
-reviewer is requested.
-If instructions for replicating the fault are contained in the linked issue then it is OK to refer back to these.
+<!-- Include sufficient instructions for someone unfamiliar with the application to test.
+Ok to refer back to instructions in the issue.
 -->
 
-<!-- delete this if you added release notes
-*This does not require release notes* because **fill in an explanation of why**
```

Comment:
I like this clause, merged into the new next at the top or bottom so it is easier to delete the instructions. Can we reformat
```
*This does not require release notes* because fill in an explanation of why
```